script/tasks/solidity/AddTokenApprovalsToCBridgeFacetPacked.s.sol

```diff
@@ -12,6 +12,13 @@ contract DeployScript is UpdateScriptBase {
     function run() public returns (address[] memory facets) {
         address facet = json.readAddress(".CBridgeFacetPacked");
 
+        // The CBridgeFacetPacked owner is the refund wallet because we need access to trigger refunds
+        // As there is only one owner, that address also needs to execute the approvals.
+        uint256 refundPrivateKey = uint256(
+            vm.envBytes32("PRIVATE_KEY_REFUND_WALLET")
+        );
+        // TODO: Check if refundPrivateKey is available, fail otherwise
```

Comment:
@0xDEnYO please recommend how to check, log, fail here (or if you prefer to do that somewhere else)

Reply:
update